.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
   "editor.tabSize": 2,
   "files.exclude": {
     "**/*.js": {"when": "$(basename).ts"},
-    "**/*.d.ts": {"when": "$(basename).ts"},
+    "src/**/*.d.ts": {"when": "$(basename).ts"},
```

Comment:
Because we only auto-generate the `*.d.ts` files in `src/` (not in `test/`)  

The old code exclude `typings/*.d.ts` which are not auto-generated.